pilot/pkg/networking/v1alpha3/util.go

```diff
@@ -57,14 +63,14 @@ func buildAddress(ip string, port uint32) core.Address {
 
 // getByAddress returns a listener by its address
 // TODO(mostrowski): consider passing map around to save iteration.
-func getByAddress(listeners []*xdsapi.Listener, addr string) *xdsapi.Listener {
-	for _, listener := range listeners {
-		if listener.Address.String() == addr {
-			return listener
-		}
-	}
-	return nil
-}
+//func getByAddress(listeners []*xdsapi.Listener, addr string) *xdsapi.Listener {
```

Comment:
feel free to delete if it's unused

Reply:
I need it when I do the management port listeners..